clients/fossid-webapp/src/test/kotlin/FossIdClientNewProjectTest.kt

```diff
@@ -107,8 +101,10 @@ class FossIdClientNewProjectTest : StringSpec({
 
     "Scans for project can be listed when there is no scan" {
         service.listScansForProject("", "", PROJECT_CODE) shouldNotBeNull {
-            checkResponse("list scans")
-            toList<Scan>() should beEmpty()
+            checkResponse("list scans", true)
+            data shouldNotBeNull {
+                values should beEmpty()
```

Comment:
IMO it's a bit odd to only check the values here (even if it implies that the keys, and therefore the whole map, is empty, too). I'd prefer `this should beEmpty()`.